### DISCUSSION TITLE
lantern团队你们好，请问你们什么时候能解决连接节点乱跳的问题？？？

### DISCUSSION THREAD

**Discussion by Appear6** [getlantern/lantern-forum-cn#1001] [General]
其中的一些连接节点是稳定的，但过了一会儿就跳走了，连接节点反反复复地来回跳，大多数时候完全无法使用。为什么你们不添加一个自选节点的功能呢？可以系统自动选择节点，也可以在系统乱跳的时候让用户自行选择选择节点。

**Comment by MaxMKMax** [comment]
@Appear6 您好. 我也帮你们反馈这个功能了. 但感觉是有一些挑战所以暂时不弄吧. 我可以彻底的理解您们的通电. 我在尽最大的能力帮你们反馈吧. 非常抱歉. 谢谢您支持蓝灯!
